R/read_absmaps.R

```diff
@@ -56,9 +63,19 @@ read_absmap <- function(name = NULL,
   url <- paste0(base_url, name, ".rda")
 
   # download to temporary file
-  out_path <- tempfile(fileext = ".rda")
-  download.file(url,
-                destfile = out_path)
+  out_path <- file.path(export_dir, paste0(name, ".rda"))
+
+  if (!file.exists(out_path)) {
+    download.file(url,
```

Comment:
Would it be worth wrapping this in `tryCatch()` or `purrr::safely()` or similar, so that you can at least provide an informative error if the download fails?